src/Chainweb/HostAddress.hs

```diff
@@ -68,6 +68,7 @@ module Chainweb.HostAddress
 , portFromText
 , pPort
 , readPortBytes
+, unsafePortFromText
```

Comment:
It doesn't look like this is used.  Can we remove it?

Reply:
It is used. Look at line 334 of `TransactionGenerator.hs`.

Reply:
I think `unsafeHostAddressFromText` could be used in `TransactionGenerator.hs:334`

Reply:
I've addressed this.